src/mol-model-formats/structure/lammps-data.ts

```diff
@@ -97,10 +101,58 @@ async function getModels(mol: LammpsDataFile, ctx: RuntimeContext, unitsStyle: U
             const key = bonds.bondId;
             const order = Column.ofConst(1, bonds.count, Column.Schema.int);
             const flag = Column.ofConst(BondType.Flag.Covalent, bonds.count, Column.Schema.int);
+            /*
             const pairBonds = IndexPairBonds.fromData(
                 { pairs: { key, indexA, indexB, order, flag }, count: atoms.count },
                 { maxDistance: Infinity }
             );
+            IndexPairBonds.Provider.set(first, pairBonds);*/
+            // Assuming 'atoms' contains the coordinates after wrapping
+            const newBonds = [];
+            for (let i = 0; i < bonds.count; i++) {
+                const [atom1Index, atom2Index] = [indexA.value(i), indexB.value(i)];
+
+                // Here, you would ideally have access to the wrapped positions of atom1 and atom2
+                // If not, you might need to apply wrapping logic here:
+                const pos1 = [atoms.x.value(atom1Index), atoms.y.value(atom1Index), atoms.z.value(atom1Index)];
```

Comment:
The array allocation here is not necessary, can use a single array and reassign it.